paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/PaymentSheetPlaygroundActivity.kt

```diff
@@ -82,29 +89,34 @@ internal class PaymentSheetPlaygroundActivity : AppCompatActivity() {
             val playgroundSettings by viewModel.playgroundSettingsFlow.collectAsState()
             val localPlaygroundSettings = playgroundSettings ?: return@setContent
 
-            PlaygroundTheme {
-                SettingsUi(playgroundSettings = localPlaygroundSettings)
+            val playgroundState by viewModel.state.collectAsState()
 
-                AppearanceButton()
+            PlaygroundTheme(
+                content = {
+                    SettingsUi(playgroundSettings = localPlaygroundSettings)
 
-                QrCodeButton(playgroundSettings = localPlaygroundSettings)
+                    AppearanceButton()
 
-                ReloadButton(playgroundSettings = localPlaygroundSettings)
+                    QrCodeButton(playgroundSettings = localPlaygroundSettings)
 
-                val playgroundState by viewModel.state.collectAsState()
-                PlaygroundStateUi(
-                    playgroundState = playgroundState,
-                    paymentSheet = paymentSheet,
-                    flowController = flowController,
-                    addressLauncher = addressLauncher,
-                )
+                    ReloadButton(playgroundSettings = localPlaygroundSettings)
+                },
+                bottomBarContent = {
```

Comment:
Nice! Can we make the bottom bar be display all the time, so that the reload button is accessible without scrolling?